ContactDetailsApi/serverless.yml

```diff
@@ -140,6 +143,9 @@ resources:
                           - Ref: 'AWS::AccountId'
                           - 'table/ContactDetails'
 custom:
+  associateWaf:
+    name: ReferenceApiWafAcl
```

Comment:
This should be project specific like; ContactDetailsApiWafAcl

Reply:
thanks, fixed everywhere